skimage/restoration/_cycle_spin.py

```diff
@@ -155,5 +155,6 @@ def _run_one_shift(shift):
         # multithreaded via dask
         futures = [dask.delayed(_run_one_shift)(s) for s in all_shifts]
         mean = sum(futures) / len(futures)
-        mean = mean.compute(get=dask.threaded.get, num_workers=num_workers)
+        mean = mean.compute(scheduler='processes',
```

Comment:
I thought `processes` would have a higher overhead than `threads` for cases like the [demo in the docs](https://github.com/scikit-image/scikit-image/blob/2b37a31e19dc21077fcf533b2322394cb0b509dd/doc/examples/filters/plot_cycle_spinning.py).  I just timed it and it is nearly identical for either choice, though, so I think either is fine.

Do you think we should expose the `scheduler` argument to the user?

Reply:
It does have a bit of overhead, although, it seems to me to be a better choice in general.

>Do you think we should expose the scheduler argument to the user?
Actually, yes! As I understood, dask scheduler could be configured globally via YAML files, env.variables, or `dask.config.set`. That said, I'd drop this argument.

Reply:
Updated.

Reply:
Yeah, dropping this argument would let the user settings in Dask take precedence.

cc @mrocklin (in case I've missed anything)

Reply:
Yeah, I'd drop all of the arguments to this method call.  If you wanted to pass things through that would be fine, but it's generally preferred not to hard-code anything.